PowerPointLabs/PowerPointLabs/SyncLab/SyncLabShapeStorage.cs

```diff
@@ -6,22 +6,19 @@
 
 namespace PowerPointLabs.SyncLab
 {
-    public class SyncLabShapeStorage : PowerPointPresentation
+    public sealed class SyncLabShapeStorage : PowerPointPresentation
     {
 
         public const int FormatStorageSlide = 0;
 
         private int nextKey = 0;
 
-        private static SyncLabShapeStorage storage;
+        private static readonly Lazy<SyncLabShapeStorage> StorageInstance =
```

Comment:
I actually used this article as a guide: http://csharpindepth.com/Articles/General/Singleton.aspx